sdk/proto/analyzer.proto

```diff
@@ -38,6 +43,18 @@ message AnalyzeRequest {
     google.protobuf.Struct properties = 2; // the full properties to use for validation.
 }
 
+// Resource defines the view of a Pulumi-managed resource as sent to Analyzers. The properties
+// of the resource are specific to the type of analysis being performed. See the Analyzer
+// service definition for more information.
+message AnalyzerResource {
```

Comment:
Should we just reuse `AnalyzeRequest` instead of defining another message type with the same contents?